docs/src/components/page-parts/platform-integrations/QuasarIntegrations.vue

```diff
@@ -76,43 +106,113 @@ const platformIntegrationIcons = [
   mdiAndroid
 ]
 
-const integrationOptions = [
+const ecosystemParts = [
+  {
+    about: 'App Extensions that help you design your UI',
+    options: [
+      {
+        label: 'QCalendar',
+        icon: mdiCalendar,
+        path: 'github.com/quasarframework/quasar-ui-qcalendar'
+      },
+      {
+        label: 'QMarkdown',
+        icon: mdiLanguageMarkdown,
+        path: 'github.com/quasarframework/app-extension-qmarkdown'
+      },
+      {
+        label: 'QMediaPlayer',
+        icon: 'ondemand_video',
+        path: 'github.com/quasarframework/app-extension-qmediaplayer'
+      },
+      {
+        label: 'All AEs',
+        icon: 'extension',
+        path: 'quasar.dev/app-extensions/discover',
+        iconColor: 'brand-accent'
+      }
+    ]
+  },
   {
-    label: 'UI',
-    name: 'User Interface Components',
-    icon: mdiWidgets,
-    path: 'components'
+    about: 'App Extensions for your app low tech integrations',
+    options: [
+      {
+        label: 'Testing',
+        icon: mdiTestTube,
+        path: 'github.com/quasarframework/quasar-testing'
+      },
+      {
+        label: 'Apollo GraphQL',
+        icon: mdiGraphql,
+        path: 'github.com/quasarframework/app-extension-apollo'
```

Comment:
Use https://github.com/quasarframework/app-extension-apollo/tree/v2 for now, since the default branch still show the Qv1 version